### ISSUE TITLE
Crash upon loading game

### ISSUE THREAD

**Issue by TrainFanatic** [leezer3/OpenBVE#442]
Hi there. I'm pretty new to the community [one of my friends told me about this game]

The issue I am experiencing occurs when I 'start' a new game. I am still able to choose trains and routes (of which I tried to follow the instruction to install as carefully as possible). However, when I press 'start', [this](https://pastebin.com/z04Fp4VX) error pops up [I'm using Mac OSX Mojave]

I'm not exactly sure what is causing this, although I think it comes from mono itself. I have tried using different versions of Mono, and installing XQuartz 2.7.11 (which is what one instruction guide told me to do). After running out of options, I have resorted to putting it as an issue.

## System specs and stuff

Mac OSX Mojave [10.14.6] running on a 2017 MacBook Pro
Memory: 8GB 2133 MHz LPDDR3 (yes i'm using a laptop)
Graphics: Intel Iris Plus Graphics 640 1536 MB

## How to replicate

Load  any version of Mono above 5.14 and XQuartz. Install Trains and Routes and press start

I'm sorry if this was caused by an error in installation. Have a nice day!

**Comment by leezer3** [comment]
First, an apology-
It's incredibly difficult for me to debug on a Mac, as I don't have a 'real' one.

First, have you tried 1.7.0.3-
https://vps.bvecornwall.co.uk/OpenBVE/Stable/OpenBVE-1.7.0.3.dmg

This is the last version I have definite confirmation of working on a real Mac.

Having said that-
Your error doesn't appear to be related to the (potetial) issue with newer versions which was graphics driver related, and would appear to be Mono itself dying.

The build server (Travis CI) is using Mono 6.8.0, so if 1.7.0.3 doesn't work I'd try that as a starting point-
https://download.mono-project.com/archive/6.8.0/macos-10-universal/MonoFramework-MDK-6.8.0.105.macos10.xamarin.universal.pkg
XQuartz shouldn't be required- Just the Mono install.

**Comment by TrainFanatic** [comment]
Hi there,

Unfortunately, despite trying out your suggestions, I am still experiencing the same problem.

I have tried diagnosing it myself and I feel like it's problem with Mono itself, but I'm not confident enough to submit an issue to mono.

I do have one question, does OpenBVE require mono to be runnable on Terminal, or does it only require the framework? My homebrew is faulty and I have tried to contact them already, not to mention that this computer has been tinkered with so much over the years that it might not even be working properly anymore.

Thanks for replying, anyways. Have a nice day!
